examples/configuration.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: pkg.crossplane.io/v1
+kind: Configuration
+metadata:
+  name: platform-ref-aws
+spec:
+  package: xpkg.upbound.io/upbound/platform-ref-aws:v0.7.0
```

Comment:
change to new repo url

Reply:
@haarchri thanks a lot for the catch, updated!